tests/specs/nodeRewardsProgram.spec.js

```diff
@@ -0,0 +1,150 @@
+import WalletFixture from '../data/wallet';
+
+function mockFetch(returnValue, type) {
+    spyOn(window, 'fetch').and.returnValue(Promise.resolve({
+        [type]: Promise.resolve(returnValue)
+    }));
+}
+
+describe('NodeRewardsProgram service tests', function() {
```

Comment:
Have you run this test in the browser?

Hint: Open the file `build/tests/start.html` in the browser.